cmd/controller/main.go

```diff
@@ -68,9 +73,12 @@ func main() {
 		"If set the metrics endpoint is served securely")
 	flag.BoolVar(&enableHTTP2, "enable-http2", false,
 		"If set, HTTP/2 will be enabled for the metrics and webhook servers")
-	opts := zap.Options{
-		Development: true,
-	}
+	//zap log config
+	flag.BoolVar(&development, "zap-devel", true, "Enable development mode")
+	flag.StringVar(&encoder, "zap-encoder", "json", "Log encoder to use (json|console)")
+	flag.StringVar(&logLevel, "zap-log-level", "info", "Level to configure the verbosity of logging. Can be one of 'debug', 'info', 'error'")
+	flag.StringVar(&stacktraceLevel, "zap-stacktrace-level", "error", "Level at and above which stacktraces are captured (one of 'info', 'error' or 'panic')")
+	opts := zap.Options{}
```

Comment:
I think the options are ok, but we are not doing anything with the options. We have to configure the logger with the correct options including the log level.

Reply:
Hi @rphillips If we agree that the options are useful. 
I’ve currently used the default options when the flags aren’t configured which will be  development and having: encoder=consoleEncoder,logLevel=Debug,stackTraceLevel=Warn

Reply:
I'd like to follow https://github.com/kubernetes-sigs/kueue/blob/main/cmd/kueue/main.go as an example.